server/main.py

```diff
@@ -11,7 +11,7 @@
 from fastapi import FastAPI, WebSocket, WebSocketDisconnect
 from pydantic import BaseModel
 
-from .codes import RoomNotFoundError, StatusCode
+from .errors import RoomNotFoundError, StatusCode
```

Comment:
`StatusCode` should still come from `.codes` not from `.errors`

Reply:
Good catch! I didn't notice it because in `errors.py` I was importing `StatusCode`, so it didn't raise an `ImportError`.